src/EditorFeatures/CSharpTest/CodeActions/IntroduceVariable/IntroduceVariableTests.cs

```diff
@@ -4073,7 +4073,7 @@ void M()
     {
         int a = 1;
         int {|Rename:y1|} = C.y;
-        var t = (a, x: y1);
+        var t = (a: a, x: y1);
```

Comment:
It just seems a bit weird to me since we added a language feature to remove some ceremony so shortly after removing the language feature.